getting_started/workflow/assets/importing_scenes.rst

```diff
@@ -224,7 +224,7 @@ FPS
 
 Most 3D export formats store animation timeline in seconds instead of frames. To ensure
 animations are imported as faithfully as possible, please specify the frames per second
-used to edit them. Failing to do this may result in minimal jitter.
+used to edit them. Failing to do this may result in a small amount of jitter.
```

Comment:
"Jitter" isn't really countable so it's strange to me to speak about "an amount of jitter". I'd remove the qualifier altogether.

Reply:
What about "perceptible jittering"?

Reply:
Actually what about "may result in shaky animations"? I have a feeling that jitter isn't going to translate well, and it's a kinda awkward word in English anyway.

Reply:
Sounds good.